doc/advanced_configuration.rst

```diff
@@ -100,16 +113,28 @@ include these lines to include these links::
 
         <div style='clear:both'></div>
 
-The file you are including is referenced with its relative path to your defined
-directory and includes all the module specific location. In this case
-``numpy.linspace``.
+Auto documenting your API with links to examples
+------------------------------------------------
 
-That will be rendered as
+The previous feature can be automated for all your modules combining
+it with the standard sphinx extension `autosummary
+<http://sphinx-doc.org/ext/autosummary.html>`. First enable it in your
+``conf.py`` extensions list.
 
-.. include:: modules/generated/numpy.linspace.examples
-.. raw:: html
+.. code-block:: python
 
-        <div style='clear:both'></div>
+    import sphinxgallery
+    extensions = [
+        ...
+	'sphinx.ext.autodoc',
+        ]
+
+Then append to your template files for classes and functions::
+
+    .. include:: {{module}}.{{objname}}.examples
+    .. raw:: html
+
+	<div class="clearer"></div>
```

Comment:
Compared to the previous solution "style='clear:both'", does that work magically or do you need some CSS to make it work ?